pyjobshop/Model.py

```diff
@@ -112,15 +113,15 @@ def add_job(
 
     def add_machine(
         self,
-        breaks: list[tuple[int, int]] | None = None,
+        breaks: Breaks | None = None,
         no_idle: bool = False,
         *,
         name: str = "",
     ) -> Machine:
         """
         Adds a machine to the model.
         """
-        machine = Machine(breaks, no_idle, name=name)
+        machine = Machine(breaks or [], no_idle, name=name)
```

Comment:
```suggestion
        if breaks is None:
            breaks = []

        machine = Machine(breaks, no_idle, name=name)
```

Nit: using `my_list or []` is an antipattern (https://stackoverflow.com/a/13710667/13184241). I try to avoid it unless it really harms readability. The common way is to set the default list using an if statement.